src/components/Manager/CommunityManagerDetails.js

```diff
@@ -26,26 +26,30 @@ const CommunityManagersDetails = () => {
 
   return (
     <>
-      <main className= "flex justify-center  items-between mt-[150px]">
-        <div className= " flex bg-gradient-lite-grey w-[1000px] justify-between rounded-2xl px-[50px] " >
-        <div className= "pt-6 mr-2">
-          <h2 className="text-[#031B4E]  font-[600] text-[34px]">Welcome Back, Emmanuel!</h2>
-          <p className="text-grey-dark text-opacity-3 font-normal text-lg pt-1">You have 8 new members!</p>
+      <main className="flex justify-center  items-between mt-[150px]">
+        <div className=" flex bg-gradient-lite-grey w-[1000px] justify-between rounded-2xl px-[50px] ">
+          <div className="pt-6 mr-2">
+            <h2 className="text-[#031B4E]  font-[600] text-[34px]">
+              Welcome Back, Emmanuel!
+            </h2>
+            <p className="text-grey-dark text-opacity-3 font-normal text-lg pt-1">
+              You have 8 new members!
```

Comment:
I believe @istoga is currently working on this in her own PR. So I didn't want to touch anything beyond moving the avatar to the correct position and I think with my VSCode setting it may have changed the spacing @annonymousauthority